enforcer-parent/enforcer/src/main/resources/Dockerfile

```
@@ -57,6 +57,7 @@ RUN \
     addgroup -S -g ${MG_USER_GROUP_ID} ${MG_USER_GROUP} \
     && adduser -S -u ${MG_USER_ID} -h ${MG_USER_HOME} -G ${MG_USER_GROUP} ${MG_USER} \
     && mkdir ${MG_USER_HOME}/lib && mkdir -p ${MG_USER_HOME}/logs \
+    && mkdir -p ${MG_USER_HOME}/lib/dropins \
```

```suggestion
    && mkdir -p ${MG_USER_HOME}/lib/dropins && mkdir -p ${MG_USER_HOME}/logs \
```
We can replace `mkdir ${MG_USER_HOME}/lib` with `mkdir -p ${MG_USER_HOME}/lib/dropins`

---

fixed